WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailLikesView.swift

```diff
@@ -157,34 +157,38 @@ private extension ReaderDetailLikesView {
     }
 
     struct SummaryLabelFormats {
-        static let onlySelf = NSLocalizedString("You_ like this.",
+        static let onlySelf = NSLocalizedString("_You_ like this.",
                                                 comment: "Describes that the current user is the only one liking the post.")
-        static let singularWithSelf = NSLocalizedString("You and %1$d blogger_ like this.",
+        static let singularWithSelf = NSLocalizedString("_You and %1$d blogger_ like this.",
```

Comment:
I was thinking. Since we know that the number for this is guaranteed to be `1`, do we still need to use format specifier here? Could we just use `_You and 1 blogger_ like this.`?

Reply:
Sure enough. I'll change it.

Reply:
According to common L10n guidelines, you should always use the placeholder though if you need numeric value (unless you spell the number out).

> If you use a numeric in the format string, as in 1 file remaining for English, it may not be localized when the user changes the region (for example, if the number set changes). Instead, use the format specifier, as in %d file remaining; otherwise, spell out the number, as in One file remaining.
_(Source: [Strings File Format reference](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/StringsdictFileFormat/StringsdictFileFormat.html), amongst others)_

One of the reason is because the way numeric values are rendered may depend on the user's locale and region settings. For example, there are two types of Arabic Numerals: Eastern Arabic Numerals ( ٠ ١ ٢ ٣ ٤ ٥ ٦ ٧ ٨ ٩ – in RTL) and Western Arabic Numerals (0, 1, 2, 3, 4, 5, 6, 7, 8, 9 – the ones we all know in our western culture). So using a `%1$d` placeholder allows iOS to use the right numeral according to the user's locale/region settings, rather than forcing one system over the other on the user. 🤷

Reply:
oh TIL. thank you for sharing the reference! I'll note this and propose a change to use the format specifier later on. 🙂

Reply:
Side note: another option, as said in that source, is to spell out the text instead of using a numeric value. For example, `_You and another blogger_ like this`.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/16984.